### ISSUE TITLE
chartGPT with RAD-NeRF

### ISSUE THREAD

**Issue by boolw** [ashawkey/RAD-NeRF#20]
I'm trying to drive the model with the text that chartGPT replies. But it has a long calling path:

text->TTS->wav->ASR->npy(logits/text)->RAD-NeRF

Such conversions seem inefficient and redundant.

Is there a better way to make it simpler and more efficient? ?

text->[???]->RAD-NeRF

For example, is it possible to use the Mel Spectrum output from TTS's am model to train or test the model? ?

**Comment by ashawkey** [comment]
This is reasonable since there are works using spectrum as input, but may need some experiments to verify.

**Comment by aishoot** [comment]
Nice work. I'm also trying it.

**Comment by exceedzhang** [comment]
good idea!
